pkg/api/types.go

```diff
@@ -1205,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
```

Comment:
I'm on the fence.  Local is local to SOMETHING.  Node is arbitrary.  There exist multi-attached hardware devices that can be local to more than one machine (but one at a time).

What happens when we encounter such a beast - is LocalVolume suddenly obsolete?

I don't want to force this idea, since we don't have generalized topo elsewhere yet, but I do want to make sure we think it through properly.  What are concrete concerns?